t/04-template-handling.t

```diff
@@ -30,10 +22,10 @@ Log::Log4perl::init(\$LedgerSMB::Sysconfig::log4perl_config);
 
 
 
-$LedgerSMB::Sysconfig::tempdir = 't/var';
+$LedgerSMB::Sysconfig::tempdir .= 't/var';
```

Comment:
Why can't we just use the temp dir? I mean, why do we need to create `/tmp/t/var` ??